src/pages/events.tsx

```diff
@@ -2,7 +2,7 @@ import { CGLEvent, parseProperties, queryDatabase } from '@/api/notion';
 import Card from '@/components/Card';
 
 import Navbar from '@/components/Navbar';
-import { getPastEvents, getUpcomingEvents, sortEvents } from '@/utils/dateUtils';
```

Comment:
[Question] Why was this import removed, was it not being used?

Reply:
yes, this is what fixed the deployment (removing unused import), I've renamed the commit to make the changes clearer!